keyboards/keychron/c1/keymaps/default/keymap.c

```diff
@@ -55,59 +63,58 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-                {   _______,               _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              KC_NO,                _______,              KC_NO,                KC_NO,                KC_NO   },
-                {   _______,               KC_NO,                _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              _______,              KC_NO,                _______,              KC_NO,                RGB_SPI,              KC_NO   },
-                {   _______,               KC_LALT,              KC_LGUI,              KC_NO,                KC_NO,                KC_NO,                _______,              KC_NO,                KC_NO,                KC_NO,                _______,              _______,              _______,              _______,              RGB_RMOD,             RGB_SPD,              RGB_MOD }
-              }
+    /*  Row:        0          1          2          3          4          5          6          7          8          9          10         11         12         13         14         15         16   */
+    [3] = {     {   RESET,     KC_BRID,   KC_BRIU,   KC_TASK,   KC_FLXP,   RGB_VAD,   RGB_VAI,   KC_MPRV,   KC_MPLY,   KC_MNXT,   KC_MUTE,   KC_VOLD,   KC_VOLU,   KC_NO,     KC_TRNS,   KC_TRNS,   RGB_TOG },
+                {   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS },
+                {   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS },
+                {   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_NO,     KC_TRNS,   KC_NO,     KC_NO,     KC_NO   },
+                {   KC_TRNS,   KC_NO,     KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_NO,     KC_TRNS,   KC_NO,     RGB_SPI,   KC_NO   },
+                {   KC_TRNS,   KC_LALT,   KC_LGUI,   KC_NO,     KC_NO,     KC_NO,     KC_TRNS,   KC_NO,     KC_NO,     KC_NO,     KC_TRNS,   KC_TRNS,   KC_TRNS,   KC_TRNS,   RGB_SAI,   RGB_SPD,   RGB_SAD }
```

Comment:
KC_LALT and KC_LGUI are wrongly inverted respect the base windows layer (layer 2), shouldn't be they set to kc_trans?

Reply:
Oh yeah, my bad. Will change it.